frontend/src/Metamaps/Map/index.js

```diff
@@ -38,6 +38,11 @@ const Map = {
   init: function () {
     var self = Map
 
+    // prevent right clicks on the main canvas, so as to not get in the way of our right clicks
+    $('#wrapper').on('contextmenu', function (e) {
```

Comment:
good, but we actually want to revert it to '#center-container' instead of '#wrapper'